lib/puppet/type/wls_datasource/connectionreservetimeoutseconds.rb

```diff
@@ -0,0 +1,12 @@
+newproperty(:connectionreservetimeoutseconds) do
+  include EasyType
+
+  desc 'The number of seconds after which a call to reserve a connection from the connection pool will timeout.'
+
+  defaultto '10'
```

Comment:
yeah,

The defaults on the wls type properties should not be there and lead to a lot of changes plus you never know if oracle will change its defaults in a release.

I will do that and add the back to default behaviour to wls_setting. basically the problem is I have a only one script for the create and modify of the wls type which does it all.  I also want to support a property reset when you specified it once and want to rollback it without knowing what the default is.